src/main/java/uk/ac/ebi/subs/validator/core/validators/ReferenceValidator.java

```diff
@@ -2,13 +2,16 @@
 
 import lombok.NonNull;
 import lombok.RequiredArgsConstructor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Service;
 import uk.ac.ebi.subs.data.component.AbstractSubsRef;
 import uk.ac.ebi.subs.repository.model.DataType;
 import uk.ac.ebi.subs.validator.data.SingleValidationResult;
 import uk.ac.ebi.subs.validator.data.structures.SingleValidationResultStatus;
 import uk.ac.ebi.subs.validator.data.structures.ValidationAuthor;
 import uk.ac.ebi.subs.validator.model.Submittable;
+import uk.ac.ebi.subs.validator.schema.JsonSchemaValidatorListener;
```

Comment:
There was not any change in this file other than the added import statements, so I guess we don't need them

Reply:
Good point, I've fixed it